tests/test_number_line.py

```diff
@@ -7,6 +7,7 @@ def test_unit_vector():
     """Check if the magnitude of unit vector along
     the NumberLine is equal to its unit_size."""
     axis1 = NumberLine(unit_size=0.4)
-    axis2 = NumberLine(width=12, x_min=-2, x_max=5)
+    axis2 = NumberLine(x_range=[-2, 5, 1])
+    axis2.width = 12
```

Comment:
I'm not sure if this was intentional, but `NumberLine`'s length can be adjusted via the `length` parameter
```suggestion
    axis2 = NumberLine(x_range=[-2, 5, 1], length = 12)
```